lua/octo/utils.lua

```diff
@@ -1658,11 +1658,8 @@ function M.apply_mappings(kind, bufnr)
         value.desc = ""
       end
       local mapping_opts = { silent = true, noremap = true, buffer = bufnr, desc = value.desc }
-      vim.keymap.set("n", value.lhs, mappings[action], mapping_opts)
-      -- TODO: These should probably be part of the config
-      if action == "add_review_comment" or action == "add_review_suggestion" then
-        vim.keymap.set("x", value.lhs, mappings[action], mapping_opts)
-      end
+      local mode = value.mode or "n"
+      vim.keymap.set(mode, value.lhs, mappings[action], mapping_opts)
```

Comment:
We should mark this as a breaking change using `!` in the commit message since users (like me 🤓) who manually set `add_review_comment/suggestion` mappings in their config will no longer have their mapping work in visual mode without them explicitely adding the mode.

It's much better this way, but still confused me

Reply:
What is the format of this? I will do upon squash